oak_restricted_kernel/src/lib.rs

```diff
@@ -286,14 +287,23 @@ pub fn start_kernel(info: &BootParams) {
         report.validate().expect("attestation report is invalid");
     }
 
-    let channel = get_channel(
+    let mut channel = get_channel(
         &kernel_args,
         GUEST_HOST_HEAP.get().unwrap(),
         acpi.as_mut(),
         sev_status,
     );
 
+    // We need to load the application binary before we hand the channel over to the syscalls, which
+    // expose it to the user space.
+    info!("Loading application binary...");
+    let payload = payload::read_payload(&mut *channel)
```

Comment:
Is payload being sent as raw bytes or as a serialized proto?

Reply:
Raw bytes. No protos around.